tests/components/nextdns/test_switch.py

```diff
@@ -0,0 +1,305 @@
+"""Test switch of NextDNS integration."""
+from datetime import timedelta
+from unittest.mock import patch
+
+from nextdns import ApiError
+
+from homeassistant.components.switch import DOMAIN as SWITCH_DOMAIN
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    SERVICE_TURN_OFF,
+    SERVICE_TURN_ON,
+    STATE_OFF,
+    STATE_ON,
+    STATE_UNAVAILABLE,
+)
+from homeassistant.helpers import entity_registry as er
+from homeassistant.util.dt import utcnow
+
+from . import SETTINGS, init_integration
+
+from tests.common import async_fire_time_changed
+
+
+async def test_switch(hass):
```

Comment:
Not compulsory, but it helps when reviewing tests (applies to all test definitions):
```suggestion
async def test_switch(hass: HomeAssistant) -> None:
```